### PR TITLE
html frame completed with some dummy data ready to be styled on

### PR DESCRIPTION
**Pull request by jakefroeb**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Testing Instructions

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
```
git fetch --all
git checkout jhf-html
serve
```

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <head>
+        <meta charset="utf-8">
+        <title>Liquid Echoes Holiday Road</title>
+        <link rel="stylesheet" href="./styles/main.css">
+    </head>
+
+    <body>
+        <main id="container">
+            <header class="header">
+                <img src="https://i.pinimg.com/originals/33/b8/69/33b869f90619e81763dbf1fccc896d8d.jpg" alt="logo">
```
Comment (SubtleCo) [inline index.html]: + id="logo"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                    </div>
+                    <div class="weatherCard">
+                        <p>day5</p>
+                        <p>temp</p>
+                        <p>forecast</p>
+                    </div>
+                </article>
+                <article class="preview__eatery">
+                    <h3>Eatery Name</h3>
+                    <button id="showDetails" value="eateryId">details</button>
```
Comment (SubtleCo) [inline index.html]: ids should be specific for event listeners, I.e. "showEateryDetails" etc

**Review by SubtleCo — APPROVED** [review]
dope

**Review by nch66862 — APPROVED** [review]
Dope

**Pull request merged**